### PR TITLE
expose MODULUS as a global string value that can be used by clients of the package

### PR DESCRIPTION
**Pull request by roberto-bayardo**
I was trying to use the MODULUS value in an init() function within prysm and noticed it had value "0" since type "Fr" in kilic mode becomes 0 when set to MODULUS.   Thus in this change I expose the modulus as a string which is more portable.

### CODE REVIEW

**Comment by Inphi** [comment]
This shouldn't happen as Go guarantees init functions are called once. Could it be that your program is linking multiple go-kzgs with different build flags?

File: bls/globals.go [hunk 1]
```diff
@@ -11,7 +11,14 @@ func ToFr(v string) (out Fr) {
 	return
 }
 
+func init() {
+	initGlobals()
+}
+
 func initGlobals() {
+	if !EqualZero(&MODULUS) {
```
Comment (Inphi) [inline bls/globals.go]: This shouldn't happen as Go guarantees init functions are called once. Could it be that your program is linking multiple go-kzgs with different build flags?

**Comment by roberto-bayardo** [comment]
@Inphi ping on this in case you missed it.

File: bls/globals.go (hunk 1 above)
Comment (roberto-bayardo) [reply bls/globals.go]: Right, but this is called initGlobal() and it's called from outside this file.

Comment (roberto-bayardo) [reply bls/globals.go]: Oh I see what you are saying, technically all init() functions should be called before the init functions in packages that import this one.  Hmm, let me take another look.

Comment (Inphi) [reply bls/globals.go]: Right. Every bignum implementation have mutually exclusive build tags. So only one `init` function in the `bls` package should be defined and called. Are you seeing issues in prysm?

Comment (roberto-bayardo) [reply bls/globals.go]: looks like the issue is just that MODULUS is of type Fr, and bignum_kilic Fr value is "0" !

OK perhaps I should just expose the string as the global.

**Comment by roberto-bayardo** [comment]
ok changed this PR to simply export MODULUS_STR as a string value to avoid the issues of various Fr types.

**Review by Inphi — APPROVED** [review]

**Pull request merged**